Tutorials fixes

diraol opened the pull request:
This PR is much easily evaluated commit by commit instead of looking to the final result, since there are some changes like removing extra spaces that would hide real changes.

Commit:
[tutorials] Improving code-block highlight_language

Commit:
[tutorials] Fix and improve Dev. Env. Setup tutorial.

Remove `kytos-napps` from the development environment setup tutorial,
since now everyone should install them from the napps-server (except if
trying to change one of them, which should not be the case for our
tutorials). Also change the tutorial to consider the installation of the
projects (python-openflow, kytos-utils and kytos) requirements by the
setup script, without the need of manually installing them.

Commit:
[tutorials] Fixing reference to tutorial/of_stats NApp

Commit:
[tutorials] Removing extra spaces and reformatting paragraphs

renanrodrigo commented on tutorials/napps/development_environment_setup.rst:
```diff
@@ -138,29 +138,36 @@ Cloning existing projects
 .. code-block:: bash
 
-  $ for project in python-openflow kytos-utils kytos kytos-napps; do
+  $ for project in python-openflow kytos-utils kytos; do
```
Shall we remove kytos-napps from the dev_env?
As we are installing the projects in develop mode, having the NApps right and ready would be nice.

If we are going to remove it, there will be changes in the logs of `kytos napps list` all over the tutorials (since no NApp will be installed at that point), which need updating. There are also text instructions, like in "How to use Kytos with Mininet", that say "You probably already have them (our NApps)", and adjusting them would be nice.

diraol replied:
@renanrodrigo The tutorials are not intended for kytos-core-team developers at first, so I wouldn't expect our 'users' to modify "kytos-napps"' NApps.

This is why I've openned this issue:
https://github.com/kytos/documentation/issues/59

=)

Does it make sense?

diraol replied:
Btw, there is also this one that can be solved together with the previous one:
https://github.com/kytos/documentation/issues/58

renanrodrigo replied:
Cool.
I'm working on #58 and #59 before approving this one, as discussed.

diraol replied:
Tks!

renanrodrigo committed:
Move instructions on RRD install to stats tutorial

Fix #58

renanrodrigo committed:
Add instructions to install NApps in the dev_env

Fix #59
NApps are no more installed from source, but from the NApps Server using kytos-utils
